manager/metrics/collector.go

```diff
@@ -0,0 +1,101 @@
+package metrics
+
+import (
+	"context"
+
+	"strings"
+
+	metrics "github.com/docker/go-metrics"
+	"github.com/docker/swarmkit/api"
+	"github.com/docker/swarmkit/manager/state/store"
+)
+
+var (
+	nodesMetric metrics.LabeledGauge
+)
+
+func init() {
+	ns := metrics.NewNamespace("swarmkit", "", nil)
+	nodesMetric = ns.NewLabeledGauge("nodes", "The number of nodes", metrics.Total, "state")
```

Comment:
Fixed here: https://github.com/docker/go-metrics/pull/11

Let me know if that works!